onnx/reference/ops/op_dft.py

```diff
@@ -1,117 +1,107 @@
 # Copyright (c) ONNX Project Contributors
 
 # SPDX-License-Identifier: Apache-2.0
-
-
-from typing import Sequence
+from __future__ import annotations
 
 import numpy as np
 
 from onnx.reference.op_run import OpRun
 
 
-def _fft(x: np.ndarray, fft_length: Sequence[int], axis: int) -> np.ndarray:
```

Comment:
Maybe worth adding a line in the PR description to summarize this change. I was surprised to see the reference implementation change, when the scope is adding version converter. I guess this is simplifying the implementation?

Reply:
Never mind, I guess this is just code from the other PR.